database/java/2015/5258.yaml

```diff
@@ -0,0 +1,20 @@
+cve: 2015-5259
+title: "Spring Social Core: CSRF
+"
+description: >
+    When authorizing an application against an OAuth 2 API provider, Spring Social is vulnerable to a 
+    Cross-Site Request Forgery (CSRF) attack.
+cvss_v2: 4.0
+references:
+    - http://pivotal.io/security/cve-2015-5258
+    - https://blog.srcclr.com/spring-social-core-vulnerability-disclosure/
```

Comment:
Maybe we should add http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-5259 here.